### PR TITLE
Switch branc to master

### PR DESCRIPTION
**Pull request by adgad**
/cc @Financial-Times/next-myft 

* All the stuff from the "switch" branch
* Load "enabled" on page load
* Client still to use our proxy
* Move `this.initialised=true` outside the async callback so things dont get loaded multiple times
* Update tests

### CODE REVIEW

File: src/myft-client.js [hunk 1]
```diff
@@ -1,49 +1,47 @@
...
+const fetchres = require('fetchres');
 
 const lib = {
 	personaliseUrl: require('./lib/personalise-url')
 };
 
 class MyFtClient {
-	constructor (opts) {
-		if (!opts.apiRoot) {
+	constructor ({apiRoot} = {}) {
```
Comment (phamann) [inline src/myft-client.js]: Nice ES6 foo!

File: src/myft-client.js [hunk 2]
```diff
@@ -55,9 +53,23 @@ class MyFtClient {
...
+	fetchJson (method, endpoint, data) {
+		var options = {
+			method,
+			headers: this.headers,
+			credentials: 'include'
+		};
+
+		if (method !== 'GET') {
+			options.body = JSON.stringify(data || {});
+		}
```
Comment (phamann) [inline src/myft-client.js]: Have we lost our `content-length` hack here?

File: src/myft-client.js [hunk 3]
```diff
@@ -77,17 +89,15 @@ class MyFtClient {
 	}
 
 	add (relationship, subject, data) {
-		this.api
-			.updateRelationship('user', this.userId, relationship, subject, data)
+		this.fetchJson('PUT', `${this.userId}/${relationship}/${subject}`, data)
```
Comment (phamann) [inline src/myft-client.js]: Can this now be a `POST`?

**Comment by phamann** [comment]
Think you'll also need to update `package.json`

**Comment by phamann** [comment]
:+1:

File: src/myft-client.js (hunk 1 above)
Comment (adgad) [inline src/myft-client.js]: As much as I'd love to take credit, this was something someone else wrote (quite possibly yourself) in `master` :(

**Pull request merged**